### ISSUE TITLE
"File has no contents" while loading

### ISSUE THREAD

**Issue by wbt** [atom/atom#20202]
When loading a file, the main file pane shows the false message "File has no contents" as an inaccurate informational point:
![image](https://user-images.githubusercontent.com/563406/70081498-af82f000-15d6-11ea-9b36-2cf23f3622fd.png)
When loading a large file, or a medium-large diff (by clicking on a file name in the Git tab; package-lock.json often does it for me), or even a small file on a slow computer, this message might be displayed for a long time, long enough that the user might believe it and wonder where the contents of the file went.   

If they leave it open to "File has no contents" for long enough (which might extend into tens of minutes or hours), the contents will eventually appear, but the informational message is telling the user the opposite of that.  They may click away to investigate long before the contents appear.

The desired behavior is that when loading, the pane says "Loading" instead of "File has no contents." If the file has finished loading, and been conclusively found to have no contents, the message should then change, but not before. 

### Versions
//atom --version
Atom    : 1.23.2924.87
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
//apm --version:
apm  2.4.3
npm  6.2.0
node 10.2.1 x64
atom 1.40.1
python
git 2.15.0.windows.1
visual studio 2015
Win10 Pro

**Comment by WilSnoop** [comment]
```text
Atom    : 1.41.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0
```

Consider updating your software:
 https://github.com/atom/atom/releases/

Please return to tell us if anything changed.

**Comment by WilSnoop** [comment]
```text
// atom --version
Atom    : 1.41.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0

// apm --version
apm  2.4.3
npm  6.2.0
node 10.2.1 x64
atom 1.41.0
```

Your Atom version looks strange.
Did you use an install from ..?
 https://github.com/atom/atom/releases/

**Comment by wbt** [comment]
@WilSnoop I agree the version looks strange, but yes.   However, this has been an issue for a while now, going back several versions.

**Comment by darangi** [comment]
@wbt, I can't seem to reproduce this error on atom `1.41.0`, can you confirm this exists on atom `1.41.0` ?

**Comment by wbt** [comment]
@darangi Do you know if there are good informational resources about what all customizations get lost in reinstalling/upgrading?

**Comment by WilSnoop** [comment]
@wbt : See if the following is helpful -
 https://discuss.atom.io/t/error-while-installing-atom/69666/2

**Comment by wbt** [comment]
I do still occasionally experience this issue.  Things tend to load faster than before, but better distinction between "loading" and "no contents" is still desirable.
